tests/Kiota.Builder.Tests/Writers/Java/CodeClassDeclarationWriterTests.cs

```diff
@@ -0,0 +1,78 @@
+    {
+        private const string defaultPath = "./";
+        private const string defaultName = "name";
+        private readonly StringWriter tw;
+        private readonly LanguageWriter writer;
+        private readonly CodeClassDeclarationWriter codeElementWriter;
+        private readonly CodeClass parentClass;
+
+        public CodeClassDeclarationWriterTests() {
+            codeElementWriter = new CodeClassDeclarationWriter(new JavaConventionService());
+            writer = LanguageWriter.GetLanguageWriter(GenerationLanguage.Java, defaultPath, defaultName);
+            tw = new StringWriter();
+            writer.SetTextWriter(tw);
+            var root = CodeNamespace.InitRootNamespace();
+            parentClass = new (root) {
+                Name = "parentClass"
+            };
+            root.AddClass(parentClass);
+        }
+        public void Dispose() {
+            tw?.Dispose();
+        }
+        [Fact]
+        public void WritesSimpleDeclaration() {
+            codeElementWriter.WriteCodeElement(parentClass.StartBlock as CodeClass.Declaration, writer);
+            var result = tw.ToString();
+            Assert.Contains("public class", result);
+        }
+        [Fact]
+        public void WritesImplementation() {
+            var declaration = parentClass.StartBlock as CodeClass.Declaration;
+            declaration.Implements.Add(new (parentClass){
+                Name = "someInterface"
+            });
+            codeElementWriter.WriteCodeElement(declaration, writer);
+            var result = tw.ToString();
+            Assert.Contains("implements", result);
+        }
+        [Fact]
+        public void WritesInheritance() {
+            var declaration = parentClass.StartBlock as CodeClass.Declaration;
+            declaration.Inherits = new (parentClass){
+                Name = "someInterface"
+            };
+            codeElementWriter.WriteCodeElement(declaration, writer);
+            var result = tw.ToString();
+            Assert.Contains("extends", result);
+        }
+        [Fact]
+        public void WritesImports() {
+            var declaration = parentClass.StartBlock as CodeClass.Declaration;
+            declaration.Usings.Add(new (parentClass) {
+                Name = "Objects",
+                Declaration = new(parentClass) {
+                    Name = "java.util",
+                    IsExternal = true,
+                }
+            });
+            declaration.Usings.Add(new (parentClass) {
+                Name = "project.graph",
+                Declaration = new(parentClass) {
+                    Name = "Message",
+                }
+            });
+            codeElementWriter.WriteCodeElement(declaration, writer);
+            var result = tw.ToString();
+            Assert.Contains("import", result);
+            Assert.Contains("project.graph.Message", result);
+            Assert.Contains("java.util.Objects", result);
```

Comment:
Make this change to test that format is correct for import statements:

            Assert.Contains("import project.graph.Message", result);
            Assert.Contains("import java.util.Objects", result);

Reply:
done